modules/claims_api/config/routes.rb

```diff
@@ -59,6 +59,12 @@
     end
   end
 
+  namespace :v2, defaults: { format: 'json' } do
+    mount OkComputer::Engine, at: '/healthcheck'
+
+    get '/veteran-identifier', to: 'veteran_identifier#show'
```

Comment:
i feel relatively strongly we should completely get away from sending anything in via headers that is not the authentication token.

that being said, i think this needs to be a post with the 4 Veteran values in the payload then.

that being said, if we're worried about it not being semantic, we could always use Google's "custom verb" concept and do something like this:
```
/veteran-identifier:find
```
or 
```
/veteran-identifier:search
```

Reply:
Switched to a POST [here](https://github.com/department-of-veterans-affairs/vets-api/pull/6845/commits/c416a5de634252f96c147b77693ae5a71af5b2e7).

Reply:
how do you feel about using something like `/veteran-id:search` or `/veteran-id:find` ? 

`/veteran-id` being a post just kind of sucks because its not creating a identifier or anything, but it cant be a get with the data we're passing in be in the path

Reply:
Eh, it's prob due to a lack of familiarity with that pattern, but I don't really like it.  It doesn't feel very "RESTy" to me.

Reply:
a post that doesnt create something isnt super restful either though :(

https://cloud.google.com/apis/design/custom_methods is doc on this colon concept

Reply:
this might be an idea for a route we could be agree on
https://stackoverflow.com/a/18933902/3013472

essentially the endpoint "creates a search" so in this case it would be something like
```
POST   /veteran-id/searches
```
that returns either the simple `id` we have now, or the concept shown in that stackoverflow link is to also return information about the search, so something like this maybe
```
{
  terms: {
    ssn: 'fsfds',
    firstName: 'fsdfdsf',
    lastName: 'fsdgfdgdf',
    birthdate: 'fdsgdfg'
  },
  result: {
    id: '21332432432'
  } 
}
```
^^^^ response structure might be a bit overkill though

Reply:
Alright, you wore me down.
Custom method it is.  😂 
[Changed Here](https://github.com/department-of-veterans-affairs/vets-api/pull/6845/commits/b133d9f2bd7591480bc9d243e80e7cd1a4c3ac75)

Reply:
😢 but 💯